platform/ios/src/MLNMapView+IBAdditions.h

```diff
@@ -1,6 +1,5 @@
 #import <Foundation/Foundation.h>
-
-@class MLNMapView;
```

Comment:
extending a forward declared class isn't allowed, i think this worked before by things being automatically imported?